mono/metadata/handle.h

```diff
@@ -516,12 +519,21 @@ This is why we evaluate index and value before any call to MONO_HANDLE_RAW or ot
 
 #endif
 
-// Get ((type)handle)->field as a handle.
+// Get handle->field as a type-handle.
 #define MONO_HANDLE_NEW_GET(TYPE,HANDLE,FIELD) (MONO_HANDLE_NEW(TYPE,MONO_HANDLE_SUPPRESS (MONO_HANDLE_RAW (MONO_HANDLE_UNSUPPRESS (HANDLE))->FIELD)))
 
+#define MONO_HANDLE_FIELD_NEW MONO_HANDLE_NEW_GET
+
 // Get handle->field, where field is not a pointer (an integer or non-managed pointer).
 #define MONO_HANDLE_GETVAL(HANDLE, FIELD) MONO_HANDLE_SUPPRESS (MONO_HANDLE_RAW (MONO_HANDLE_UNSUPPRESS (HANDLE))->FIELD)
 
+#define MONO_HANDLE_FIELD MONO_HANDLE_GETVAL
```

Comment:
IMO this should mention`VAL` in the name somewhere so that it's evident that it should only be used on non-managed fields.

Reply:
MONO_HANDLE_FIELD_VAL or MONO_HANDLE_FIELD_VALUE?
I was going for brevity..and FIELD isn't the C++ term. Maybe leave things alone here.

Reply:
I guess "VAL" is the nomenclature for non-managed-pointer and VALUE almost starts to sound like, hey, just the value of the field, no matter the type.